INSTALL.md

```diff
@@ -64,25 +59,27 @@ Next steps after installation:
      .\env\Scripts\activate
      ```
 
-2. Install `scikit-learn-intelex`:
+### Install from PyPI Channel (recommended by default)
+
+Install `scikit-learn-intelex`:
 
    ```bash
    pip install scikit-learn-intelex
    ```
 
-#### 📦 Supported configurations for PyPI <!-- omit in toc -->
+#### 📦 Supported Configurations for PyPI <!-- omit in toc -->
 
 | OS / Python version | **Python 3.8** | **Python 3.9** | **Python 3.10** | **Python 3.11** |
```

Comment:
we have 3.12 supported already. 

+ might be same info can be put in more compact form? now it's same cpu, gpu state for all components

Reply:
@napetrov done